### PR TITLE
Documentation

### PR DESCRIPTION
**Pull request by scento**
This pull request tries to reduce the lack of operational documentation by providing two step-for-step guides to get a development environment for 1327.

Despite of this, it would be nice, if a [changelog](http://keepachangelog.com/en/0.3.0/), a feature description and a notice related to the WIP-state of the repository would be kept.

At first, I intended to write in the Wiki, but it seemed to be closed for foreign contributions.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,5 +4,47 @@
...
+```
+
+### Virtual Environment
+
+Another way of executing this django application is the use of a virtual python environment. This way bypasses the needs for a virtual machine and simplifies the life with multiple python versions installed:
+
+```bash
+virtualenv -p /usr/bin/python3 env
+source env/bin/activate
+pip install -r requirements.txt
```
Comment (Bartzi) [inline README.md]: please also add `pip install -r requirements-test.txt`

Comment (scento) [inline README.md]: Done.

**Comment by Bartzi** [comment]
I think having a changelog is a good idea once we are ddone with the initial release, for now I think this is not very helpful.
A note on the WIP state is indeed a good idea ;) maybe you want to add that to your PR, too?

**Comment by scento** [comment]
I added the note regarding the WIP state. Everything should be fine now. :)

**Comment by Bartzi** [comment]
Nice

**Pull request merged**